### ISSUE TITLE
syft.frameworks.torch.tensors.interpreters.native.Tensor.shape BUG

### ISSUE THREAD

**Issue by wentaiwu92** [OpenMined/PySyft#2576]
**Describe the bug**
When attempting to call .shape on a Syft tensor contained in a BaseDataset, it always return torch.Size([0]).

**To Reproduce**
```
t = torch.tensor([1.,2.,3.])
t.shape                         # torch.Size([3])
bds = sy.BaseDataset(t, t)
bds.send(bob)
bds.data.shape                  # expected torch.Size([3]), but got torch.Size([0])
```

**Desktop (please complete the following information):**
 - OS: win10
 - Version: Pytorch 1.1.0, PySyft 0.1.24a1

**Comment by LaRiffle** [comment]
That's strange, I have torch.Size([3])
Maybe update the syft package and rerun this snippet

**Comment by wentaiwu92** [comment]
Solved after bumping up to 0.1.26a1. :)
